IdentityServer/v7/DPoP/WebClient/CustomProofService.cs

```diff
@@ -1,26 +0,0 @@
-using Duende.AccessTokenManagement;
-
-namespace WebClient;
-
-public class CustomProofService : DefaultDPoPProofService, IDPoPProofService
-{
-    public CustomProofService(IDPoPNonceStore dPoPNonceStore, ILogger<DefaultDPoPProofService> logger) : base(dPoPNonceStore, logger)
-    {
-    }
-
-    public new Task<DPoPProof> CreateProofTokenAsync(DPoPProofRequest request)
-    {
-        if (request.Url.StartsWith("https://localhost:5005"))
-        {
-            // this shows how you can prevent DPoP from being used at an API endpoint
-            // and this will cause the request to send the same token with the Bearer scheme
-            //return Task.FromResult<DPoPProof>(null);
-        }
-        return base.CreateProofTokenAsync(request);
```

Comment:
I think this is, while perhaps unusual, a valid use case. Maybe we need to follow this idea all the way through to the end, but I don't really want to just toss this. Are users confused?